llvm/include/llvm/Transforms/Strakhov2/Strakhov2.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#define DEBUG_TYPE "Strakhov2"
```

Comment:
@iastrakhov, please remove this macro from header to avoid redefinition. It seems even unused for this pass